marklogic-data-hub-central/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.tsx

```diff
@@ -37,11 +37,20 @@ const CreateEditMappingDialog = (props) => {
       setSrcQuery(props.mapData.sourceQuery);
       setSelectedSource(props.mapData.selectedSource);
       if(props.mapData.selectedSource === 'collection'){
-      let srcCollection = props.mapData.sourceQuery.substring(
-          props.mapData.sourceQuery.lastIndexOf("[") + 2, 
-          props.mapData.sourceQuery.lastIndexOf("]") - 1
-      );
-      setCollections(srcCollection);
+      if(props.mapData.sourceQuery.includes('[') && props.mapData.sourceQuery.includes(']')) {
+          let srcCollection = props.mapData.sourceQuery.substring(
+              props.mapData.sourceQuery.lastIndexOf("[") + 2,
+              props.mapData.sourceQuery.lastIndexOf("]") - 1
+          );
+          setCollections(srcCollection);
+      }
+      else{
+          let srcCollection = props.mapData.sourceQuery.substring(
+            props.mapData.sourceQuery.lastIndexOf("(") + 2,
+            props.mapData.sourceQuery.lastIndexOf(")") - 1
+          );
+          setCollections(srcCollection);
+      }
```

Comment:
@AkshayBajajML do we need this code block here if the issue is taken care of by handling it in conversionFunctions?

Reply:
Change in conversion function takes care of the collection name for mapping card that we see by default after deploying the sample project. This highlighted block change takes care of the collection name when we make any changes to existing source query name by editing it.